src/libappimage/type2.c

```diff
@@ -65,7 +65,11 @@ void appimage_type2_extract_regular_file(sqfs* fs, sqfs_inode* inode, const char
     squash_extract_inode_to_file(fs, inode, target);
 }
 
-bool appimage_type2_resolve_symlink(sqfs* fs, sqfs_inode* inode) {
+bool appimage_type2_resolve_symlink(sqfs* fs, sqfs_inode* inode, uint hops_remaining) {
```

Comment:
`hops_remaining` is the wrong name for that variable, it creates a wrong impression of what the variable is used for, and also creates the impression that networking would be used for this somehow. Please try to find a more suitable name.

Reply:
networking... !? I guess that the name is really bad if you got that impression.

Reply:
"hops" are a term in routing, not on the filesystem.

Reply:
Changed by `depth`.